shared/src/state-migrations/20-add-simulation-properties.ts

```diff
@@ -0,0 +1,28 @@
+import { seededRandomState } from '../simulation';
+import type { UUID } from '../utils';
+import type { Migration } from './migration-functions';
+
+export const addSimulationProperties20: Migration = {
+    actions: null,
```

Comment:
You have to migrate the `[SimulatedRegion] Add simulated region` action too, since the state you're migrating may be the initial state with all actions re-applied on import.